README.md

```diff
@@ -1 +1,145 @@
-# Budget-app
\ No newline at end of file
+# Expense Tracking App
+
+## Getting Started
+
+This repository contains ruby files with ruby code.
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Expense Tracking App <a name="about-project"></a>
+
+**Expense Tracking App** The Recipe app keeps track of all your expenses. It will allow you to add an expense categories, and under each of these catagories it helps you keep track of how much you spent, what you spent on and the total amount of each expense categories.  Also, since allows you to choose a nice looking icon for your expense category.
+
+## 🛠 [Live Preview](https://budget-app-f5vg.onrender.com/)
```

Comment:
- [ ] Please make sure to update your live demo version with the different changes and don't forget the back button as requested by previous reviewers. Right now it not working as expected facing an issue like you see on the screenshot below ⬇️ 
![Capture](https://github.com/kalbek/Budget-app/assets/6615394/83448d76-c966-48ee-8f9b-f0d44e982680)